api/datadoghq/v1alpha2/datadogpodautoscaler_types.go

```diff
@@ -119,6 +145,31 @@ type DatadogPodAutoscalerApplyPolicy struct {
 	ScaleDown *common.DatadogPodAutoscalerScalingPolicy `json:"scaleDown,omitempty"`
 }
 
+// DatadogPodAutoscalerFallbackApplyPolicy defines how recommendations should be applied in fallback mode.
+type DatadogPodAutoscalerFallbackApplyPolicy struct {
+	// Mode determines recommendations that should be applied by the controller:
+	// - Apply: Apply all fallback recommendations.
+	// - Preview: Recommendations are generated, but the controller will not apply them.
+	// - Disabled: Recommendations are not generated.
+	// +optional
+	// +kubebuilder:default=Apply
+	Mode DatadogPodAutoscalerFallbackMode `json:"mode"`
+
+	// ErrorWindowSeconds defines the time window the controller will wait after detecting an error before applying recommendations.
+	// +optional
+	// +kubebuilder:default=600
+	// +kubebuilder:validation:Minimum=180
+	// +kubebuilder:validation:Maximum=1200
+	ErrorWindowSeconds int32 `json:"errorWindowSeconds,omitempty"`
```

Comment:
Should we ask users to distinguish these cases? I think the goal is to trigger if we have no data for X seconds